charts/ocis/values.yaml

```diff
@@ -102,7 +101,7 @@ store:
   type: nats-js
   # -- Nodes of the store to use.
   nodes:
-    - "{{ .appNameNats }}:9233"
```

Comment:
Seems like your YAML formatter introduces a lot of changes to this YAML. What kind of YAML formatter are you using? I'm using https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml

We can change the YAML format, but should do it in a separat PR and document which formatter we are using (+ adding a CI step to ensure the format)